mtb-wifi-mw-manifest.xml

```diff
@@ -1096,4 +1101,33 @@
       </version>
     </versions>
   </middleware>
+  <middleware>
+    <name>ota-update</name>
+    <id>ota-update</id>
+    <uri>http://git-ore.aus.cypress.com/repo-staging/ota-update</uri>
```

Comment:
Needs to be changed to:
<uri>https://github.com/Infineon/ota-update</uri>

I have tried modifying, but I do not have permissions on this PR.
I have pulled the PR to be local and made the change but I cannot push it back (permissions).
MR for the manifest repo here:
https://gitlab.intra.infineon.com/repo/mtb-wifi-mw-manifest/-/merge_requests/182